node/cmd/guardiand/node.go

```diff
@@ -1003,10 +1003,16 @@ func runNode(cmd *cobra.Command, args []string) {
 		if idx != 0 {
 			// try to connect to guardian-0
 			for {
+				// tilt uses this hostname format
 				_, err := net.LookupIP("guardian-0.guardian")
 				if err == nil {
 					break
 				}
+				// load tests use this hostname format
+				_, err = net.LookupIP("guardian-0")
+				if err == nil {
+					break
+				}
 				logger.Info("Error resolving guardian-0.guardian. Trying again...")
```

Comment:
I don't know if this needs to be changed?  I could go either way.

Reply:
I don't think so, it's only in the devnet branch and the point is pretty much the same.